### DISCUSSION TITLE
카카오 인앱에서 제 계정이 아닌 타인의 계정으로 로그인 하여 매크로를 돌릴수 있나요?

### DISCUSSION THREAD

**Discussion by ZER0ADD** [kimytsc/covid-rest-vaccine-macro#78] [Q&A] [answered]
부모님 잔여백신 접종을 해드리려고 하는데 이 코드를 직접 실행하시기에는 무리가 있습니다. 하여 제 핸드폰을 통해 대신 잡아드리려고 하는데 제 폰에 있는 카카오 인앱으로 부모님의 계정에 로그인해서 잡아드릴수 있을까요?

**Comment by kimytsc** [comment, marked as answer]
이 부분은 테스트를 조금 해봐야 할 것 같습니다. 로그인 상태가 변경이 되는건지를 봐야 할 것 같은데, 만약 로그인 이후 노출되는 이름이 부모님의 성함이라면, 스크립트를 돌려도 괜찮지만 아니라면 돌릴 수 없을 것 같습니다.

**Reply by ZER0ADD** [reply]
카카오는 불가능 한것 같네요 ㅠㅜ 네이버로 다시 도전해 보겠습니다 !

**Reply by kimytsc** [reply]
테스트를 해본 결과, 현재로써는 카카오톡 인앱 브라우저에서는 카카오 로그인을 하더라도 연동되지 않고 카카오톡 인증된 계정으로만 연동되는 것으로 파악되었습니다.

네이버쪽으로 시도해보시는게 좋을 것 같습니다.

**Comment by thebluegate** [comment]
안되더라구요.저도 부모님꺼 제 폰에서 해보려했는데 안되요

**Reply by ZER0ADD** [reply]
그러게요 아쉽게도 네이버만 시도해야겠네요

**Reply by kimytsc** [reply]
테스트를 해본 결과, 현재로써는 카카오톡 인앱 브라우저에서는 카카오 로그인을 하더라도 연동되지 않고 카카오톡 인증된 계정으로만 연동되는 것으로 파악되었습니다.

네이버쪽으로 시도해보시는게 좋을 것 같습니다.
